runtime/oti/j9.h

```diff
@@ -350,7 +350,6 @@ static const struct { \
 #define IS_LTYPE(firstChar) ('L' == (firstChar))
 
 #define J9_IS_STRING_DESCRIPTOR(str, strLen) (((strLen) > 2) && (IS_REF_OR_VAL_SIGNATURE(*(str))) && (';' == *((str) + (strLen) - 1)))
-
```

Comment:
No need to include this file in this PR, only an empty line change in this file.

Reply:
I'm not using `IS_REF_OR_VAL_SIGNATURE` because it only checks the first character of the string that is passed to it. `bcvIsReferenceTypeDescriptor` is used to check whether a string is a normal class name (like `MyClass`)  or an equivalent reference type descriptor (like `LMyClass;`). If I only checked the first character then class names that began with L or Q would be seen as reference type descriptors even if they were not.